### PR TITLE
Add VY Docs

### PR DESCRIPTION
**Pull request by froboy**
[Review this PR with GitPod](https://gitpod.io/#https://github.com/YCloudYUSA/yusaopeny_docs/pull/7)

Migrate docs from:
- https://community.openymca.org/c/resources-and-training-for-content-editors/virtual-ymca-docs/21
- Daxko Barcode - https://community.openymca.org/t/virtual-y-daxko-barcode-authentication/825
- ReClique SSO - pdf from YUSA

### CODE REVIEW

**Comment by froboy** [comment]
@podarok for your review.

File: content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+1. Save your settings.
+1. Set Daxko Barcode as your main authorization plugin at the Virtual YMCA settings: `/admin/openy/openy-gc-auth/settings`.
+
+Once enabled, the module enables granular configuration to messages that users will receive on the page. It allows changing "Barcode" to something different, like "Member ID", and allows adding help text to assist members in finding their ID. It also allows for global help text to direct members to help channels in case they're unable to log in.
+
+![The Daxko Barcode configuration form](./daxko-barcode-config.png)
+
+Once the module is enabled, members will be presented with the appropriately titled field to log in to Virtual Y.
+
+![The Virtual Y login page with Daxko Barcode authentication and ReCAPCHA enabled](./daxko-barcode-login.png)
```
Comment (podarok) [inline content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md:23]:
```suggestion
![The Virtual Y login page with Daxko Barcode authentication and reCAPCHA enabled](./daxko-barcode-login.png)
```

Comment (podarok) [reply content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md:23]: see https://www.google.com/recaptcha/about/

File: content/en/docs/user-documentation/virtual-ymca/resources.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- [<i class="fa fa-video" title="Video"></i> Southeast Ventura approach to Virtual Y](https://www.youtube.com/watch?v=ukhUDhGZNp0&t=352s&ab_channel=OpenY%20)
+- [<i class="fa fa-file-powerpoint"></i> Y North set up](https://drive.google.com/drive/folders/17JrSpIlZ50AygJRq0xZQ_73-aVksyVWs?usp=sharing)
+- [<i class="fa fa-file-powerpoint"></i> West Cook's Virtual Y Plan](https://drive.google.com/file/d/1XvgsA7B47EBU1x4LTBGrbp8thSbHU9F8/view?usp=sharing)
+- [<i class="fa fa-file-powerpoint"></i> Austin's Virtual Y Experience Map](https://drive.google.com/file/d/1tX2xFZPcj25aWvAMKdS1e8GDURYEkTue/view?usp=sharing)
+- [<i class="fa fa-globe" title="Web page"></i> Y North Gear Guide ](https://www.ymcanorth.org/live-streaming-gear-guide)
+
+# Further Learning
+
+- [<i class="fa fa-globe" title="Web page"></i> Y-USA Virtual Y FAQs](https://ycloud.y.org/y-cloud-faq) 
+- [<i class="fa fa-globe" title="Web page"></i> Virtual Y sandboxes](https://virtual-y-sandboxes.openy.org/)
```
Comment (podarok) [inline content/en/docs/user-documentation/virtual-ymca/resources.md:36]:
```suggestion
- [<i class="fa fa-globe" title="Web page"></i> Virtual Y sandboxes](https://virtual-y-sandboxes.y.org/)
```

File: content/en/docs/user-documentation/virtual-ymca/software-requirements.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* Open Y version 2.0 or newer.
+* The ability to install Virtual YMCA modules
+
+## Virtual Y Standalone Site 
+### Hosting 
+**Tech stack required**
+
+* Ubuntu Server (local or Cloud environment) with 2CPU and 2GB of RAM minimum.
+*  Server configured with LAMP stack (Linux/Apache/MySQL/PHP).
+* PHP 7.3+
```
Comment (podarok) [inline content/en/docs/user-documentation/virtual-ymca/software-requirements.md:15]:
```suggestion
* PHP 7.4+ ( PHP 8.1+ is recommended)
```

**Review by podarok — CHANGES_REQUESTED** [review]
Small things to be approved by @froboy

File: content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md (hunk 1 above)
Comment (froboy) [reply content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md:23]: Thanks. I can never remember how to get that right.

Comment (podarok) [reply content/en/docs/user-documentation/virtual-ymca/authentication-providers/daxko-barcode/_index.md]: Nobody can, that's why we have code review in place )))

File: content/en/docs/user-documentation/virtual-ymca/resources.md (hunk 2 above)
Comment (froboy) [reply content/en/docs/user-documentation/virtual-ymca/resources.md:36]: Works, yay!

**Pull request merged**